reader.go

```diff
@@ -501,9 +507,11 @@ func decodeLineOfMediaPlaylist(p *MediaPlaylist, wv *WV, state *decodingState, l
 	case !state.tagProgramDateTime && strings.HasPrefix(line, "#EXT-X-PROGRAM-DATE-TIME:"):
 		state.tagProgramDateTime = true
 		state.listType = MEDIA
-		if state.programDateTime, err = time.Parse(DATETIME, line[25:]); strict && err != nil {
+
+		if state.programDateTime, err = TimeParse(line[25:]); strict && err != nil {
 			return err
 		}
+
```

Comment:
This is not me, it is my formatter in Sublime Go :)
but sure i can remove it through console editor